MHZ19.cpp

```diff
@@ -39,12 +49,75 @@ int MHZ19::getAccuracy()
 {
 	if (_result == MHZ19_RESULT_OK)
 	{
-		int value = static_cast<int>(_response[4]);
+		int value = static_cast<int>(_response[5]);
 		return value;
 	}
 	return _result;
 }
 
+void MHZ19::setAutoCalibration(bool mode)
+{
+	if (mode) {
+		sendCommand(0x79,0xA0,0x00,0x00,0x00,0x00);
+	} else {
+		sendCommand(0x79,0x00,0x00,0x00,0x00,0x00);
+	}
+
+}
+
+MHZ19_RESULT MHZ19::setAccuracy(MHZ19_MEASUREMENT _acc)
+{
+	switch (_acc) {
+		  case PPM1000:
+			sendCommand(0x86,0x00,0x00,0x00,0x03,0xE8);
+			break;
+		  case PPM2000:
+			sendCommand(0x86,0x00,0x00,0x00,0x07,0xD0);
+			break;
+		  case PPM3000:
+			sendCommand(0x86,0x00,0x00,0x00,0x0B,0xB8);
+			break;
+		  case PPM5000:
+			sendCommand(0x86,0x00,0x00,0x00,0x13,0x88);
+ 			break;
+		  case PPM10000:
+			sendCommand(0x86,0x00,0x00,0x00,0x27,0x10);
+ 			break;
+		  default:
+			sendCommand(0x86,0x00,0x00,0x00,0x07,0xD0);

```

Comment:
Probably it is better to do nothing by default, just do not calibrate sensor at all.